index.js

```diff
@@ -70,8 +64,8 @@ const title =  {
     title.state = state; 
     title.update();
   },
-  setCharacterLength: length => {
-    title.charLength = length
+  setCharacterLength: async length => {
+    title.charLength = await storage.getBytesInUse();
```

Comment:
Nice! Was going to say we should move to async/await from promises. If we do want to support this in all browsers, we may have to compile that however.

Reply:
It's a chrome extension ;)

Reply:
I wasn't sure if you wanted to move towards other browsers with `localStorage`. Or was that just for the githack?

Reply:
Ah yeah, that's just for githack previews